### ISSUE TITLE
Feature request, support PDFjs url rewrite

### ISSUE THREAD

**Issue by prototype23** [DevExpress/testcafe-hammerhead#1802]
One method to view a pdf with PDFjs in a website, is to use an iframe  like this:

`<iframe src="http://my.site/pdfjs/web/viewer.html?file=http%3A%2F%2Fmy.site%2Fmyfiles%2Fmypdf.pdf"></iframe>`

Iframe url, targets the PDFjs viewer and viewer opens the file defined on the parameter `file`.
For more details you can read [here ](https://github.com/mozilla/pdf.js/wiki/Setup-pdf.js-in-a-website).

Testcafe is rewriting that url like 
`<iframe src="http://10.10.23.53:50642/GkN25mFKQ!i/http://my.site/pdfjs/web/viewer.html?file=http%3A%2F%2Fmy.site%2Fmyfiles%2Fmypdf.pdf" src-hammerhead-stored-value="http://my.site/pdfjs/web/viewer.html?file=http%3A%2F%2Fmy.site%2Fmyfiles%2Fmypdf.pdf"></iframe>`
missing to replace the second url on the `file` param. Result is to successfully load the PDFjs viewer but
receive a cross domain error for the file.

On my previous [question ](https://github.com/DevExpress/testcafe-hammerhead/issues/1730)it was clear that we can't skip that testcafe behaviour. But can we at least improve it 😇 in order to change the second param too? 

On my tests changing the second param, like this
`?file=http://10.10.23.53:50642/GkN25mFKQ!i/http%3A%2F%2Fmy.site%2Fmyfiles%2Fmypdf.pdf`
is making PDFjs to work normally!

**Comment by miherlosev** [comment]
Hi @prototype23 

Could you please provide a complete example in which I can reproduce this behavior?

**Comment by prototype23** [comment]
@miherlosev Well, i can't reproduce the issue on a clean example. Here is one: https://github.com/prototype23/Testcafe-PDFjs-example

Any idea why url proxy is not replacing the url param? I already tried these:
- Installed the same testcafe versions. 
- Simplified the test with the problem (removed Request hooks from the test with the problem)
- Checked the html structure to be the same.
- Checked the timing where js sets the iframe src. On my test with the problem, src is set on the modal's ready callback. Here the src is set instantly. I added a timeout to simulate this on this test but still i can't reproduce the problem.
